android-components/components/browser/state/src/main/java/mozilla/components/browser/state/engine/middleware/SessionPrioritizationMiddleware.kt

```diff
@@ -16,13 +22,21 @@ import mozilla.components.concept.engine.EngineSession.SessionPriority.HIGH
 import mozilla.components.lib.state.Middleware
 import mozilla.components.lib.state.MiddlewareContext
 import mozilla.components.support.base.log.logger.Logger
+import mozilla.components.support.base.coroutines.Dispatchers as MozillaDispatchers
 
 /**
  * [Middleware] implementation responsible for updating the priority of the selected [EngineSession]
  * to [HIGH] and the rest to [DEFAULT].
+ *
+ * @property updatePriorityAfterMillis Update priority to default after timeout.
  */
-class SessionPrioritizationMiddleware : Middleware<BrowserState, BrowserAction> {
+class SessionPrioritizationMiddleware(
+    private val updatePriorityAfterMillis: Long = 300000,
```

Comment:
nit: maybe we could create a constant for this value instead of a magic number and explain what it does and its default value's meaning with a comment

Reply:
Good point, I added a comment for the number